docs/source/conf.py

```diff
@@ -42,7 +42,7 @@
     'sphinxcontrib.mermaid'
 ]
 # autosummary_generate = True
-doctest_path = [os.path.abspath('../../bim2sim/')]
+doctest_path = [os.path.abspath('../../bim2sim/'), os.path.abspath('../utilities/plugindiagram')]
```

Comment:
Did this had any impact? So did it work?

Reply:
no, it has not. I will check for it working on issue #535